metrics/cli/test/cli/doctor/factory/doctor_factory_test.dart

```diff
@@ -34,13 +40,19 @@ void main() {
       firebaseService: firebaseService,
       sentryService: sentryService,
     );
+    final dependencies = Dependencies(dependenciesMap);
```

Comment:
✅ 
Would it be easier to use the mock implementation of the `Dependencies` class in tests?